src/legacy/core_plugins/timelion/server/series_functions/graphite.test.js

```diff
@@ -17,32 +17,31 @@
  * under the License.
  */
 
-import proxyquire from 'proxyquire';
```

Comment:
These changes are derived from the [NP upgrade that was never backported to 6.8](https://github.com/elastic/kibana/commit/4473f1e54710b94a3d4099eaf3a252f1524343aa#diff-b91231d41b9f65cb69cbe22f9a344f22)